app/models/container_image.rb

```diff
@@ -82,5 +82,14 @@ def has_compliance_policies?
     !plist.blank?
   end
 
+  def annotate_deny_execution(causing_policy)
```

Comment:
Based on [this error message](https://github.com/ManageIQ/manageiq/pull/7536/files#diff-b65b69c5edaddcf64ef4a666f394489eR859), this call is only available for OpenShift images.  This should really be implemented on an OpenShift implementation of `ContainerImage`.

But, since `ContainerImage does not yet support STI, add an error handler here in case someone calls this method directly.  Stopping calls via `MiqAction` seems insufficient to protect this method.